Emby.AutoOrganize/Model/FileOrganizationResult.cs

```diff
@@ -5,12 +5,20 @@ namespace Emby.AutoOrganize.Model
 {
     public class FileOrganizationResult
     {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="FileOrganizationResult"/> class.
+        /// </summary>
+        public FileOrganizationResult()
+        {
+            DuplicatePaths = new List<string>().AsReadOnly();
```

Comment:
```suggestion
            DuplicatePaths = new List<string>();
```
We just assume people use it correctly, `AsReadOnly` adds a lot of overhead when accessing the collection.